### DISCUSSION TITLE
Apps not importing

### DISCUSSION THREAD

**Discussion by Roland0909** [Micke-K/IntuneManagement#243] [Ideas]
Hej Micke

I was trying to export / import Apps.
However it doesn't seem to work.

Is there a problem with it and/or how do I check it?

**Comment by Micke-K** [comment]
Hello,

Not really fully supported. There is no API for exporting the files. They are encrypted and encryption keys sre not available. There are a few things you can do. Export works. Import works if you have the original intunewin file eg in the same folder as the json file. 

There are a few issues/discussions regarding this. There are some ways of exporting the intunewin file as well but that requires that the encryption information is available. 

Cheers!
